builder/amazon/ebs/builder.go

```diff
@@ -187,6 +187,9 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 			Debug:                             b.config.PackerDebug,
 			EbsOptimized:                      b.config.EbsOptimized,
 			ExpectedRootDevice:                "ebs",
+			HttpEndpoint:                      b.config.HttpEndpoint,
```

Comment:
These changes should be added to the other ebs builders as well. The steps and `RunConfig` are shared between them.

Reply:
Propagated these additions to the other builders.